Cython/Compiler/PyrexTypes.py

```diff
@@ -5746,4 +5747,4 @@ def get_all_subtypes(tp, seen=None):
             else:
                 for sub_tp in subtype_or_iterable:
                     get_all_subtypes(sub_tp, seen)
-    return list(seen)
+    return list(seen.values())
```

Comment:
Hmm, you probably meant `.keys()`, but the lack of test failures suggests that this is unused in the current tests?
```suggestion
    return list(seen)
```